ui/lib/kv/addon/components/page/secret/metadata/version-diff.hbs

```diff
@@ -1,11 +1,90 @@
-<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @pageTitle={{@model.name}}>
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @pageTitle={{@path}}>
   <:tabLinks>
     <LinkTo @route="secret.details" data-test-secrets-tab="Secret">Secret</LinkTo>
     <LinkTo @route="secret.metadata.index" data-test-secrets-tab="Metadata">Metadata</LinkTo>
-    <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Metadata">Version History</LinkTo>
-    <LinkTo @route="secret.metadata.diff" data-test-secrets-tab="Metadata">Version Diff</LinkTo>
+    <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Version History">Version History</LinkTo>
+    <LinkTo @route="secret.metadata.diff" data-test-secrets-tab="Version Diff">Version Diff</LinkTo>
   </:tabLinks>
 
+  <:toolbarFilters>
+    {{! left side version selector }}
+    <BasicDropdown @class="popup-menu" @horizontalPosition="auto-right" @verticalPosition="below" as |D|>
+      <D.Trigger data-test-version-dropdown class="toolbar-link {{if D.isOpen ' is-active'}}">
+        Version
+        {{this.leftSideVersion}}
+        <Chevron @direction="down" @isButton={{true}} />
+      </D.Trigger>
+      <D.Content @defaultClass="popup-menu-content">
+        <nav class="box menu" aria-label="version-left">
+          <ul class="menu-list">
+            {{#each @metadata.sortedVersions as |versionData|}}
+              <li>
+                <button
+                  type="button"
+                  class="link"
+                  {{on "click" (fn this.selectVersion versionData.version D.actions "left")}}
+                  disabled={{(or versionData.destroyed versionData.deletion_time)}}
+                >
+                  Version
+                  {{versionData.version}}
+                  {{#if (eq versionData.version this.leftSideVersion)}}
```

Comment:
I'm a little confused - are the icons supposed to give us information about the version state? or the diff state?

Reply:
Version state for destroyed and deleted, but if you've selected the version is has the green icon (circle not filled in).
![image](https://github.com/hashicorp/vault/assets/6618863/958b570c-541a-4bd0-9fe7-1e0b064f016a)

Reply:
Update: modified to match version history and version drop down. Added a is-active link for selected versions.